go.mod

```diff
@@ -3,17 +3,15 @@ module go.infratographer.com/permissions-api
 go 1.20
 
 require (
-	github.com/ThreeDotsLabs/watermill v1.2.0
 	github.com/authzed/authzed-go v0.8.0
 	github.com/authzed/grpcutil v0.0.0-20230524151342-4caf7fd1108a
 	github.com/labstack/echo/v4 v4.11.1
-	github.com/nats-io/nats.go v1.27.1
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.16.0
 	github.com/stretchr/testify v1.8.4
-	go.infratographer.com/x v0.3.4
+	go.infratographer.com/x v0.3.5-0.20230804152936-1fb9cfe07da6
```

Comment:
Are we going to be on this prerelease commit for everything until v0.3.5 is cut?